ksp_plugin/pile_up.cpp

```diff
@@ -415,15 +451,19 @@ void PileUp::DeformPileUpIfNeeded(Instant const& t) {
     CHECK(Contains(apparent_part_rigid_motion_, part));
   }
 
-  using ApparentPileUp = Frame<enum class ApparentPileUpTag, NonRotating>;
+  Time const Δt = t - psychohistory_->back().time;
```

Comment:
Clear the PID if Δt changes.

Reply:
Done in the `PID`, but with a huge tolerance because I don't trust KSP/Unity to be super-precise.  What we are after is physics warp anyway.